r/tests/testthat/test-compute-aggregate.R

```diff
@@ -351,3 +351,71 @@ test_that("value_counts", {
   expect_identical(as.data.frame(value_counts(a)), result_df)
   expect_identical(as.vector(value_counts(a)$counts), result_df$counts)
 })
+
+test_that("any.Array", {
+  
+  data <- c(1:10, NA, NA)
+  array_data <- Array$create(data)
+  
+  expect_equal(as.vector(any(array_data > 5)), any(data > 5))
+  expect_equal(as.vector(any(array_data < 1)), any(data < 1))
+  expect_equal(as.vector(any(array_data < 1, na.rm = TRUE)), any(data < 1, na.rm = TRUE))
```

Comment:
These tests could be simplified by using the helper function `expect_vector_equal` (which is defined in `helper-expectation.R`). E.g.:
```suggestion
  expect_vector_equal(any(input > 5), data)
  expect_vector_equal(any(input < 1), data)
  expect_vector_equal(any(input < 1, na.rm = TRUE), data)
```

Reply:
Thanks, updated!